container/features/src/main/resources/features-minion.xml

```diff
@@ -28,4 +28,39 @@
       <feature>minion-core-api</feature>
       <bundle>blueprint:mvn:org.opennms.features/org.opennms.features.discovery/${project.version}/xml/blueprint-discovery-distPollerDaoMinion</bundle>
     </feature>
+    
+    <!-- TrapD feature -->
+    <feature name="opennms-events-traps" description="OpenNMS :: Features :: Events :: Trap Daemon" version="${project.version}">
```

Comment:
Please move this feature into the features.xml file and give it this name and descr:
```
<feature name="opennms-trapd" description="OpenNMS :: Trapd" version="${project.version}">
```